frontend/src/lib/logic/scenes/useAttachedLogic.ts

```diff
@@ -1,12 +1,15 @@
 import { BuiltLogic, Logic, LogicWrapper, beforeUnmount, useMountedLogic } from 'kea'
 import { useEffect, useState } from 'react'
 
+// Type used to attach a logic to the scene logic
+export type SceneLogicAttachTo<LogicType extends Logic = Logic> = BuiltLogic<LogicType> | LogicWrapper<LogicType>
```

Comment:
I'd rather we not do this. `LogicWrapper` and `BuiltLogic` are the two shapes a logic can take in Kea (before building and after). I personally don't understand why must we abstract away them away... And if we really want to, this has nothing to do with `AttachTo` --> I'd just call it `AnyKindOfLogic` or something like that.

Reply:
I also replaced the "fix" to preflightLogic. Your scene's logic has nothing to do with preflightLogic, so I don't see how is linking them for no reason a cleaner solution. We should fix whatever is causing storybook to setup a weird environment where the logics don't work.